packages/cli/src/commands/send-tx.ts

```diff
@@ -0,0 +1,85 @@
+import pickBy from 'lodash.pickby';
+
+import sendTx from '../scripts/send-tx';
+import { parseContractReference } from '../utils/contract';
+import ConfigVariablesInitializer from '../models/initializer/ConfigVariablesInitializer';
+import { promptIfNeeded, networksList, promptForNetwork, argsList, methodsList, proxiesList, proxyInfo, InquirerQuestions } from '../prompts/prompt';
+import { SendTxPropsParams, SendTxSelectionParams } from './interfaces';
+import promptForMethod from '../prompts/method-params';
+
+const name: string = 'send-tx';
+const signature: string = name;
+const description: string = 'Send a transaction to the specified contract instance. Provide the [address], method to call and its arguments if needed';
+
+const register: (program: any) => any = (program) => program
+  .command(signature, undefined, { noHelp: true })
+  .usage('[options]')
```

Comment:
Why we specify a contract address and a method as optional? It seems to me they are not optional.

Reply:
Good question.  My first answer would be because _some_ of the defined flags are indeed optional, but definitely that usage description is not accurate. We might check which is the `commander.js` or the general cli convention in these cases.